src/_imaging.c

```diff
@@ -121,7 +121,7 @@ typedef struct {
     ImagingAccess access;
 } ImagingObject;
 
-static PyTypeObject Imaging_Type;
+static PyTypeObject* Imaging_Type;
```

Comment:
maybe you already know, but a hint just in case: this should eventually be `HPyGlobal`